migration/migrator/migrations/course/20210320080101_add_pause_timer.py

```diff
@@ -0,0 +1,8 @@
+
+def up(config, database, semester, course):
+    database.execute('ALTER TABLE queue ADD COLUMN IF NOT EXISTS time_paused INTEGER;')
+    database.execute('ALTER TABLE queue ADD COLUMN IF NOT EXISTS time_paused_start TIMESTAMP WITH TIME ZONE;')
```

Comment:
On migrating, if there are any existing paused students, on attempting to go to the office hours queue page, they get an exception:

```
TypeError (Code: 0) thrown in /usr/local/submitty/site/app/models/OfficeHoursQueueModel.php (Line 214) by:
        return $this->current_queue_state['time_paused'];


Message:
Return value of app\models\OfficeHoursQueueModel::getCurrentTimePaused() must be of the type integer, null returned

Stack Trace:
#0 /usr/local/submitty/site/vendor/twig/twig/src/Extension/CoreExtension.php(1499): app\models\OfficeHoursQueueModel->getCurrentTimePaused()
```

We will need to set an initial value for these columns for any rows where `paused = true`. For this, it's fine to just set the `time_paused_start` to be the current now timestamp.

Reply:
This should be good to go now.